src/sql/parts/notebook/notebook.component.ts

```diff
@@ -344,10 +344,10 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		attachSelectBoxStyler(kernelDropdown, this.themeService);
 
 		let attachToContainer = document.createElement('div');
-		let attachTodropdwon = new AttachToDropdown(attachToContainer, this.contextViewService, this.modelRegistered,
+		let attachToDropdown = new AttachToDropdown(attachToContainer, this.contextViewService, this.modelRegistered,
 			this.connectionManagementService, this.connectionDialogService, this.notificationService, this.capabilitiesService);
-		attachTodropdwon.render(attachToContainer);
-		attachSelectBoxStyler(attachTodropdwon, this.themeService);
+			attachToDropdown.render(attachToContainer);
```

Comment:
There may be some weird indentation here. May be worth it to do a quick formatting pass.